subscriptionform/templates/registration/password_reset_email.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 
 {% blocktrans %}Someone asked for password reset for email {{ email }}. Follow the link below:
-{{ protocol}}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}{% endblocktrans %}
+{{ protocol}}://{{ domain }}{% endblocktrans %}{% url 'password_reset_confirm' uidb64=uid token=token %}
```

Comment:
Als de blocktrans na de url verplaatst wordt levert dat de volgende foutmelding op: 
`File "/usr/local/lib/python3.4/site-packages/django/templatetags/i18n.py", line 534, in do_block_translate
    raise TemplateSyntaxError("'blocktrans' doesn't allow other block tags (seen %r) inside it" % token.contents)
django.template.exceptions.TemplateSyntaxError: 'blocktrans' doesn't allow other block tags (seen "url 'password_reset_confirm' uidb64=uid token=token") inside it
`